gensuperset/make-t4d/generate-make-t4d.sh

```diff
@@ -2,15 +2,20 @@
 # Usage: ./generate-make-t4d.sh "apache/superset:3.1.arm" "tech4dev/superset:0.41" "output_folder"
 
 # Check if the correct number of arguments are provided
-if [ "$#" -ne 3 ]; then
-    echo "Usage: $0 <base_image> <output_image> <output_folder>"
+if [ "$#" -ne 4 ]; then
+    if [ "$#" -ne 4 ]; then
```

Comment:
this double `if` condition is a typo right

Reply:
Yes it was a typo. Removed it .